.github/workflows/build.yml

```diff
@@ -120,8 +122,19 @@ jobs:
       - uses: actions/checkout@v4
       - uses: actions/setup-python@v5
         name: Install Python ${{ matrix.python }}
+        id: python
         with:
           python-version: ${{ matrix.python }}
+          update-environment: false
+
+      - uses: astral-sh/setup-uv@v5
+        with:
+          enable-cache: false
+
+      - name: Setup environment
+        run: |
+          uv venv --python "${{ steps.python.outputs.python-path }}"
+          uv pip install pip --group test
```

Comment:
Nice, I missed this was added in 0.6.7! I was following the PR but missed the merge.